deep_qa/training/trainer.py

```diff
@@ -148,8 +150,15 @@ def __init__(self, params: Params):
             os.makedirs(parent_directory, exist_ok=True)
 
         # `model.fit()` parameters.
+        self.num_gpus = params.pop("num_gpus", 1)
```

Comment:
This needs to be added to the docstring, with some usage info (like that we will just use batch parallelism, and that it's not currently compatible with adaptive batch sizes, and increases padding lengths with dynamic padding).